rose/client/main.py

```diff
@@ -71,27 +70,23 @@ def send_message(self, msg):
         self.client.sendLine(str(msg).encode('utf-8'))
 
 
-def load_driver_module(file_path):
+def load_driver_module(driver_path):
     """
-    Load the driver module from file
+    Load the driver module from the specified path.
 
-    Arguments:
-      file_path (str): The path to the driver module
-
-    Returns:
-        Driver module (module)
-
-    Raises:
-        ImportError if the module cannot be loaded
+    :param driver_path: Path to the driver module.
+    :return: The loaded module.
+    :raises ImportError: If there's an issue loading the module.
```

Comment:
Why use the more cryptic styntax instead of the current style? Both are valid syntax for docstrings and supported by documentation tools. Lees keep the current style.

Reply:
Thanks, reverted to the original pydoc